Thirdlesson/ObjectTracking /ObjectCatch.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by s1000633 on 5/18/21.
+//
+
+#ifndef OPENCV_TUTORIALS_OBJECTCATCH_H
+#define OPENCV_TUTORIALS_OBJECTCATCH_H
+
+#include <opencv2/opencv.hpp>
+#include <iostream>
+
+class ObjectCatch {
+private:
+    cv::Mat _imgHSV;
+    cv::Mat _imgThresholded;
+    cv::Mat _res;
+public:
+    ObjectCatch(const cv::Mat& image, const cv::Scalar& lower_b, const cv::Scalar& upper_b);
+    void show_res();
```

Comment:
can be const

Reply:
what can be const?

Reply:
function show_res(). It doesn't change anything, it just prints some results. So it can be decalred as const, e.g
void show_res() const;